### ISSUE TITLE
Ability to see UI before creating an account

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#2535]
So, I would like to find an instance that is more usable for me. A lot of this is to do with background images, the size and colour of the font and the colour of the background, etc.

But in order to find a good instance, I have to create an account - just to see what the UI is like.

At the moment I don't want to move until I've found somewhere *just right*, because I can't move my posts to a new account and it's such a big disruptive thing. Which means I'm using a style script (not a beginner activity!) and I don't have the skills to edit that script to make it do what I want. It's very unsatisfactory! But I can at least read my posts, it's just... really ugly. And it doesn't meet a whole bunch of other accessibility needs.

![screen shot 2017-04-27 at 16 47 33](https://cloud.githubusercontent.com/assets/6607512/25491884/39b1b294-2b69-11e7-8a57-fa5abac14e0f.png)

I shouldn't have to try out a bunch of style scripts to make the site accessible, and I would love if I could see the user interface before signing up.

This is related, in my head, to being able to see the local timeline before signing up (#1053) - those are the two things I'd like to know about before making an account! Which is to say, I would sign up to see those two things, and then immediately abandon my account once I found out they didn't suit me.

* * * *

- [Yes] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [N/A] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by expenses** [comment]
A theme engine (#2028) to change the theme back to the default would really help with this, but I think the ability to have a glance at the local timeline is also essential (#1778).

**Comment by Cassolotl** [comment]
>to change the theme back to the default would really help with this

I imagine that *would* help a lot of people, but it wouldn't help me - I'm looking for a theme that is more usable than the default. I find the text really hard to read on the default theme!

**Comment by Cassolotl** [comment]
I made a page on the Mastodon wiki for people to upload screenshots of their instances to help others see a preview without having to register: 

When I [posted asking about this on Mastodon](https://cybre.space/@cassolotl/724868) a lot of people replied to say it was a good idea, and a few people replied (publicly and privately) to say that their age or other factors meant that they really struggle with the text and background colours. I know a lot of people are using user styles to make things easier in this way and that's not acceptable or accessible for a lot of people.

**Comment by rixx** [comment]
I'd really like to see this feature land in some way – maybe we could couple the preview of an instance's local timeline with the preview of the instance's theme?

**Comment by Cassolotl** [comment]
So, a bit of an update.

We've got [themes](https://github.com/tootsuite/mastodon/pull/4959) (changeable by users in settings) coming in the next version.

We've also got the [ability to see the federated timeline on an instance's About page](https://github.com/tootsuite/mastodon/pull/4122) before making an account or logging in.

I'd like to keep this issue open, because the default Mastodon theme will be the only theme we can be *sure* an instance has - and if the default theme isn't accessible, knowing in advance that an instance has it isn't that useful. It's the *custom* themes that are available that will be useful to know about before joining. I would need an instance to have a non-default accessible theme, and I'd like to _not_ have to join an instance to see the custom themes to see if the site is usable for me.

Having said that, I'm also aware that basically every other website on the whole entire internet requires you to register to find out if you can use it...! But maybe we can do better.

**Comment by enterprisey** [comment]
+1

(commenting so I can get back to this if I ever decide to get into masto dev)

**Comment by naught101** [comment]
@Cassolotl: Not a solution to the problem, but a work-around for sites that are hard to read is to use the [Stylus plugin](https://addons.mozilla.org/en-US/firefox/addon/styl-us/) to customise the CSS of a site. I use it to make the main column wider on my instance, but you can also change text colour and size.

**Comment by nightpool** [comment]
(deleted an off-topic comment)
